agent-identity/typescript/.env.example

```
@@ -0,0 +1,23 @@
+# Asgardeo Agent Credentials
+# ---------------------------------------
+AGENT_ID=<your-agent-id>
+AGENT_SECRET="<your-agent-secret>"
+
+# ---------------------------------------
+# Google Gemini API Key
+# ---------------------------------------
+GOOGLE_API_KEY=<google-api-key>
+
+# LLM model used by the agent (any supported model can be used).
+MODEL_NAME="gemini-2.5-flash"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)